Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scratch
+FROM alpine
 ADD dist/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
This certs should be coming from alpine then:

RUN apk --update add ca-certificates && rm -rf /var/cache/apk/*